libs/eo/transfers/src/lib/eo-transfers-table.component.ts

```diff
@@ -201,12 +204,13 @@ export class EoTransferTableComponent implements AfterViewInit {
     this.dataSource.data = this.transfers;
   }
 
-  isDateActive(date: number): boolean {
-    return new Date(date).getTime() >= new Date().getTime();
+  compensateForEndOfDay(inputDate: string): number {
+    const date = new Date(inputDate);
+    return date.setDate(date.getDate() - 1);
```

Comment:
Since the endDate can be at any point on the day, then I think you should check here, that you change the endDate if it is on midnight.

Reply:
Removed this implementation